galeracluster/source/documentation/mysql-wsrep-options.rst

```diff
@@ -183,6 +184,58 @@ You can execute the following ``SHOW VARIABLES`` statement to see how this varia
     | innodb-wsrep-applier-lock-wait-timeout | 10    |
     +----------------------------------------+-------+
 
+
+.. _`plugin_wsrep_provider`:
+.. rst-class:: section-heading
+.. rubric:: ``plugin_wsrep_provider``
+
+.. index::
+   pair: Parameters; plugin_wsrep_provider
+
+Defines optional settings the node passes to the wsrep Provider.
+
+.. csv-table::
+   :class: doc-options
+
+   "Command-line Format", "``--plugin_wsrep_provider``"
+   "System Variable", "``plugin_wsrep_provider``"
+   "Variable Scope", "Global"
+   "Dynamic Variable", ""
+   "Permitted Values", "String"
+   "Default Value", "ON"
+   "Valied Values", "ON, OFF"
+   "Initial Version", "MySQL-wsrep: 8.4.2, MariaDB: 4.x"
+
+When this option is enabled, the node loads the wsrep Provider configuration options through the ``plugin_wsrep_provider`` plugin, instead of using ``wsrep_provider_options`` options. ``wsrep_provider_options`` options can no longer be used. 
+
+The configuration required to use ``plugin_wsrep_provider``:
+
+.. code-block:: mysql
+
+   [mysqld.1]
+   wsrep-on=ON
+   wsrep-cluster-address=gcomm://
+   wsrep-provider=@ENV.WSREP_PROVIDER
+   binlog-format=ROW
```

Comment:
I wonder if we must mention this now, 8.4 has deprecated this, see https://dev.mysql.com/doc/refman/8.4/en/replication-options-binary-log.html#sysvar_binlog_format

Reply:
Removed